### PR TITLE
Adding an alt image text

### PR DESCRIPTION
**Pull request by da5nsy**
...  for the example of an alt image text showing when the actual image cannot be shown

### CODE REVIEW

**Comment by da5nsy** [comment]
Waiting on #2313, so that I can visually check the build, before marking as 'Ready for review'

File: book/website/_build/html/_sources/community-handbook/style/style-figures.md [hunk 1]
```diff
@@ -97,9 +97,8 @@ Another advantage of using alt text is when an image cannot be loaded in a brows
 ```{figure} ../../figures/alt-text-demo.png
 ---
 name: alt-text-demo
-alt: Two people happily browsing files in a drawer of documents.
+alt: A visual example of a web browser when it is not able to access an image.
```
Comment (malvikasharan) [inline book/website/_build/html/_sources/community-handbook/style/style-figures.md:100]:
```suggestion
alt: This image is used for example using a wrong image path that a web browser can't access and hene does not display correctly.
```

**Review by malvikasharan — COMMENTED** [review]
Describing the purpose of the image in style guide.

**Comment by malvikasharan** [comment]
Ok, I solved the Jupyter Book deployment error but it doesn't solve Netlify error. Letting @RaoOfPhysics know as he reviewed my PR to fix deployment issue.

**Comment by da5nsy** [comment]
> I solved the Jupyter Book deployment error
Just checking - this is a temporary fix that manually updates the build right?

**Comment by malvikasharan** [comment]
Yes, Chris will be publishing a new version of JupyterBook, which will then pulled in this repo. ✨

**Pull request closed without merging**